js/ui/camera.js

```diff
@@ -475,10 +475,6 @@ util.extend(Camera.prototype, /** @lends Map.prototype */{
             this.pitching = true;
         }
 
-        if (this.zooming && !around) {
-            around = tr.pointLocation(tr.centerPoint.add(to.sub(from).div(1 - 1 / scale)));
-        }
-
```

Comment:
Removing this makes sense. :+1: It was preventing the else statement below from ever being reached when the zooming option was specified, and only that else branch knows how to pan.

In the gl-native implementation, `anchor` is applied after `center`, not instead of `center`, and `anchor` has no default value.